### PR TITLE
Fix `first-commit-message-for-new-prs`

### PR DESCRIPTION
**Pull request by yakov116**
GitHub is going to give us a very hard time. There are almost no more class specific selectors

Test on: 
https://github.com/sindresorhus/refined-github/compare/master...yakov116:text?expand=1 (Cross-fork)
https://github.com/sindresorhus/refined-github/compare/master...sindresorhus:lint?expand=1 (Same-repo)

### CODE REVIEW

File: source/features/use-first-commit-message-for-new-prs.tsx [hunk 1]
```diff
@@ -7,16 +7,14 @@ import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
 async function init(): Promise<void | false> {
-	const commitCount = await elementReady<HTMLElement>([
-		'.overall-summary > ul > li:nth-child(1) .text-emphasized', // Cross fork
-		'[href="#commits_bucket"] .Counter' // Same repository
-	].join());
+	const commitsBucket = await elementReady<HTMLElement>('#commits_bucket');
+	const commitCount = commitsBucket?.previousElementSibling?.querySelector('ul > li:nth-child(1) .text-emphasized');
```
Comment (fregante) [inline source/features/use-first-commit-message-for-new-prs.tsx:11]: 😱 

Did you check this on both same-repo and cross-fork?

Comment (yakov116) [reply source/features/use-first-commit-message-for-new-prs.tsx:11]: Yes testing link in first post

Comment (fregante) [reply source/features/use-first-commit-message-for-new-prs.tsx:11]: Use `octicon-git-commit`

File: source/features/use-first-commit-message-for-new-prs.tsx [hunk 2]
```diff
@@ -7,16 +7,12 @@ import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
 async function init(): Promise<void | false> {
-	const commitCount = await elementReady<HTMLElement>([
-		'.overall-summary > ul > li:nth-child(1) .text-emphasized', // Cross fork
-		'[href="#commits_bucket"] .Counter' // Same repository
-	].join());
-
+	const commitCount = (await elementReady<HTMLElement>('div.Box.mb-3 .octicon-git-commit'))?.nextElementSibling;
```
Comment (fregante) [inline source/features/use-first-commit-message-for-new-prs.tsx:10]: I don’t think HTMLElement is necessary here

File: source/features/use-first-commit-message-for-new-prs.tsx (hunk 1 above)
Comment (fregante) [reply source/features/use-first-commit-message-for-new-prs.tsx]: What did you test exactly? Neither one of those commits have a description

Comment (yakov116) [reply source/features/use-first-commit-message-for-new-prs.tsx]: Your right I did not test the description part but the title changes. Will test it soon.

**Review by fregante — APPROVED** [review]
Merge after testing

**Comment by yakov116** [comment]
Screenshot_20201117-003429_Microsoft Remote Desktop

**Pull request merged**